src/components/tools/Partials/Dashboard/index.svelte

```diff
@@ -100,78 +106,45 @@
                 <div
                   class="bx--grid bx--grid--full-width bx--grid--tab-content"
                 >
-                  <div class="bx--row margin--v-16">
-                    <div class="bx--col">
-                      <slot name="tab_content_map_title"
-                        >tab_content_map_title</slot
-                      >
-                    </div>
-                  </div>
-
-                  <div class="bx--row margin--v-16">
-                    <div class="bx--col">
-                      <slot name="tab_content_slippy_map"
-                        >tab_content_slippy_map</slot
-                      >
-                    </div>
-                  </div>
-
-                  <div class="bx--row">
-                    <div class="bx--col">
-                      <slot name="tab_content_slippy_map_controls">
-                        tab_content_slippy_map_controls
-                      </slot>
-                    </div>
-                  </div>
+                  <View useTabs="{true}">
+                    <slot slot="map_title" name="map_title" />
```

Comment:
This syntax looks a little odd but it's how you forward slotted content to a child component from a parent component.